types/state_transaction.go

```diff
@@ -10,7 +10,7 @@ import (
 
 var ExecuteStateSyncABIMethod, _ = abi.NewMethod("function execute(" +
 	"bytes32[] proof, " +
-	"tuple(uint256 id, address sender, address receiver, bytes data, bool skip) stateSync)")
+	"tuple(uint256 id, address sender, address receiver, bytes data) stateSync)")
```

Comment:
Seems like test is still going to fail, until there is skip field on `StateReceiver` SC .

Reply:
I think they still need to change the names of some fields in `StateReceiver`.